### PR TITLE
Add ability to reset brush with session$resetBrush/Shiny.resetBrush

### PR DESCRIPTION
**Pull request by jcheng5**
Necessary for some linked brushing scenarios involving Crosstalk

### CODE REVIEW

File: srcjs/shinyapp.js [hunk 1]
```diff
@@ -641,19 +641,19 @@ var ShinyApp = function() {
...
+      // the side-effect of rendering the deps, singletons,
+      // and <head> still occur
+      exports.renderHtml($([]), message.content.html, message.content.deps);
+    } else {
+      targets.each(function (i, target) {
+        exports.renderContent(target, message.content, message.where);
+        return message.multiple;
+      });
+    }
+  });
```
Comment (jcheng5) [inline srcjs/shinyapp.js]: Just un-indenting by one level.

**Comment by wch** [comment]
Does this actually reset the brush and clear the div? It's not clear to me how that works.

**Comment by jcheng5** [comment]
All the image/plot divs listen for `shiny-internal:brushed`, and clear themselves when it's triggered. I added that a while back so that multiple image/plot outputs could share the same brushId.

**Comment by wch** [comment]
Oh right, now I remember.

Does this also close #1177?

**Pull request merged**